lib/models/rocket.py

```diff
@@ -17,6 +17,12 @@ class CoordinateSystemOrientation(str, Enum):
 
 
 class Rocket(BaseModel):
+
+    def __eq__(self, other):
+        if not isinstance(other, Rocket):
+            return False
+        return self.dict() == other.dict()
```

Comment:
Just to clear somethings up for me: Why the override of the equality operator? In which situations will this equality check be used?

Reply:
Truth be said this is actually a hack to make this pass:
```
mock_update_flight.assert_called_once_with(
            '123', rocket=Rocket(**stub_rocket)
        )
```

But definitely should be refactored.
I'll try to think in a non-verbose yet functional way to overcome that, thanks for bringing this.

Reply:
patched here https://github.com/RocketPy-Team/Infinity-API/pull/41/commits/8a8e68091a731b0c109c203e28c046d8a7a102b8